sdk/core/perf/inc/azure/perf/base_test.hpp

```diff
@@ -38,9 +38,12 @@ namespace Azure { namespace Perf {
     std::string m_recordId;
     std::string m_proxy;
     bool m_isPlayBackMode = false;
+    bool m_isInsecureEnabled = false;
 
     void SetTestProxy(std::string const& proxy) { m_proxy = proxy; }
```

Comment:
Documentation for `SetTestProxy`?

Reply:
PostSetUp, PreCleanUp and the other one are just going beyond the expectation :) .  SetTestProxy wants to just meets the expectation 🤣 

just kidding.  I can add something, for sure :)

Reply:
To be clear: The reason I asked this question was that I was looking at the checked in code to try to get a better understanding of the design of your changes (because clearly I was missing something important). 

And I had no idea what SetTestProxy did from the declaration because it was completely undocumented. I had to go to the implementation to understand what it did.